### DISCUSSION TITLE
iron burn on TI

### DISCUSSION THREAD

**Discussion by Chucrutty** [sunflower-land/sunflower-land#2465] [Ideas]
I believe there could be a third purchase option inside the treasure island store, one that uses 1-5 iron, and some wood.... that new item can be dug and has a 75% - 100% chance of getting something, but it can be the whole list.

![image](https://user-images.githubusercontent.com/125780525/233127062-fbe09255-4323-4814-af7b-7b7e63a869e5.png)

what you guys think?

**Comment by future2back** [comment]
Great idea to be discussed friend.

**Comment by adamhannigan** [comment]
Thanks for raising this. We are actually in the process of changing the requirements for the Sand Drill to require Iron.

Too many Sand Drills were minted last season (than we anticipated) due to the low cost. We will add some iron to those requirements.
